### DISCUSSION TITLE
:music codenonstop

### DISCUSSION THREAD

**Discussion by thienpow** [helix-editor/helix#10139] [General]
#### Describe your feature request

example config:
[music]
spotify=api_uri
itune=api_uri
autoplay_previous=true #start playing when editor is open, and start from previous ended.
show_equalizer=mini | medium | hide #mini = small animation at status bar, midium=box size at top-right/bottom-right 

example command:
:ms codenonstop 
:music codenonstop
:ms fav #this will list top10 most played music, which means, it also keep a record of playlist tracking database.
:ms channel bobchannel or :ms ch bobchannel # this will auto play from the channel without selecting a specific music

**Comment by pickfire** [comment]
@thienpow I am surprised to find you here. \o

In the meantime, can always do `:sh mpc next`. I believe something like this would be clear that it should be part of plugin.
